ZAPDTR/ZAPD/Main.cpp

```diff
@@ -188,14 +188,7 @@ int main(int argc, char* argv[])
 		}
 		else if (arg == "-eh")  // Enable Error Handler
 		{
-#if !defined(_MSC_VER) && !defined(__CYGWIN__)
-			signal(SIGSEGV, ErrorHandler);
-			signal(SIGABRT, ErrorHandler);
-#else
-			HANDLE_WARNING(WarningType::Always,
-			               "tried to set error handler, but this ZAPD build lacks support for one",
-			               "");
-#endif
+			
```

Comment:
We still want the proper error handler to work for Linux based systems. I would just comment out the warning.